pom.xml

```diff
@@ -45,52 +53,17 @@
                     </execution>
                 </executions>
             </plugin>
-
             <plugin>
-                <!-- Building an executable JAR -->
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
We don't need to create executable jar or any jar for that matter as of now.
It was necessary to run a chat server, but now we don't have it.
And SpringBoot will take care of running our app.